src/main/java/com/mathworks/ci/RunMatlabBuildBuilder.java

```diff
@@ -165,6 +175,7 @@ private void createMatlabScriptByName(FilePath uniqueTmpFolderPath, String uniqu
         final FilePath matlabCommandFile =
                 new FilePath(uniqueTmpFolderPath, uniqueScriptName + ".m");
         final String tasks = envVars.expand(getTasks());
+        final String buildOptions = envVars.expand(getBuildOptions().getOptions());
```

Comment:
This will throw if `getBuildOptions()` returnns null?

Reply:
Thanks for pointing it out! Updated the code!